app/src/main/java/app/musikus/usecase/activesession/GetOngoingPauseDurationUseCase.kt

```diff
@@ -0,0 +1,18 @@
+package app.musikus.usecase.activesession
+
+import app.musikus.utils.TimeProvider
+import app.musikus.utils.minus
+import kotlinx.coroutines.flow.first
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.seconds
+
+class GetOngoingPauseDurationUseCase(
+    private val activeSessionRepository: ActiveSessionRepository,
+    private val timeProvider: TimeProvider
+) {
+    suspend operator fun invoke() : Duration {
+        val state = activeSessionRepository.getSessionState().first() ?: return 0.seconds
+        if (state.currentPauseStartTimestamp == null) return 0.seconds
```

Comment:
This would make some things more difficult since this usecase is called every time the sessionState is updated...

An ongoing pause duration of 0s kind of means "no pause" 😄